### PR TITLE
Add Uniques to Westmint and Statemint

### PR DESCRIPTION
**Pull request by shawntabrizi**
This adds the Uniques pallet to Westmint and Statemint.

Why should Kusama steal all the fun?

### CODE REVIEW

File: polkadot-parachains/statemint-runtime/src/lib.rs [hunk 1]
```diff
@@ -331,14 +331,18 @@ impl InstanceFilter<Call> for ProxyType {
 				c,
 				Call::Balances(..)
 					| Call::Assets(..)
+					| Call::Uniques(..)
```
Comment (apopiak) [inline polkadot-parachains/statemint-runtime/src/lib.rs:334]:
```suggestion
				Call::Balances(..) | Call::Assets(..) | Call::Uniques(..)
```

File: polkadot-parachains/westmint-runtime/src/lib.rs [hunk 2]
```diff
@@ -329,14 +331,18 @@ impl InstanceFilter<Call> for ProxyType {
 				c,
 				Call::Balances(..)
 					| Call::Assets(..)
+					| Call::Uniques(..)
```
Comment (apopiak) [inline polkadot-parachains/westmint-runtime/src/lib.rs:334]:
```suggestion
				Call::Balances(..) | Call::Assets(..) | Call::Uniques(..)
```

File: polkadot-parachains/westmint-runtime/src/lib.rs [hunk 3]
```diff
@@ -234,6 +233,9 @@ impl pallet_sudo::Config for Runtime {
 	type Call = Call;
 }
 
+/// We allow root and the Relay Chain council to execute privileged asset operations.
```
Comment (apopiak) [inline polkadot-parachains/westmint-runtime/src/lib.rs:236]:
```suggestion
```

Comment (apopiak) [reply polkadot-parachains/westmint-runtime/src/lib.rs:236]: no council on westmint, right?

**Review by apopiak — APPROVED** [review]
LGTM

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**